decidim-admin/spec/helpers/moderations/reports_helper_spec.rb

```diff
@@ -13,8 +13,13 @@ module Moderations
           )
         end
 
+        let(:redesign_enabled) { false }
+
         before do
           allow(helper).to receive(:current_or_new_conversation_path_with).and_return("/conversations/1")
+          # rubocop:disable RSpec/AnyInstance
+          allow_any_instance_of(ActionView::Base).to receive(:redesign_enabled?).and_return(redesign_enabled)
```

Comment:
Could we move this to a shared context that is included in the target spec as this seems to be repeated across multiple specs?